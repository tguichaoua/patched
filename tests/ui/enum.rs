use patched::Patch;

#[derive(Patch)]
enum Foo {
    Goo(u64),
}

fn main() {}
