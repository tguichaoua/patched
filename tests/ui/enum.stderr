error: Patch does not support derive for enums
 --> tests/ui/enum.rs:4:1
  |
4 | / enum Foo {
5 | |     Goo(u64),
6 | | }
  | |_^
